### DISCUSSION TITLE
Is this a toplevel related bug in ocaml?

### DISCUSSION THREAD

**Discussion by hansonchar** [ocaml/ocaml#11977] [Q&A]
`ocaml` crashed when I tried to `#show Base.Error`.  Is this a bug?  The steps to reproduce the crash are available at https://github.com/ocaml/ocaml/issues/11976

**Comment by nojb** [comment]
Hello, friendly advice: please do not open an issue _and_ a discussion at the same time; it doesn't help to keep things organized. There has already been some feedback in the issue, so let's close this one.
